WordPress/gutenberg
# Fix fixture serialization with double newline ending

Related: #2056
Related: #2034 (cc @nylen)

This pull request seeks to...

1. Add some utilities around regenerating fixtures
2. Fix fixture generation including double newline in output

In #2034, the test for valid serialized output was updated to include a newline at the end. However, the newline appending should have been removed from the subsequent output writing. Currently there are two newlines:

https://github.com/WordPress/gutenberg/blob/cec69e8/blocks/test/full-content.js#L170
https://github.com/WordPress/gutenberg/blob/cec69e8/blocks/test/full-content.js#L175

This pull request removes the latter of these.

__Testing instructions:__

Verify that running the (newly-introduced) fixtures generate scripts result in no changes or test failures:

```
npm run fixtures:regenerate
```
